lib/containers/gene.js

```diff
@@ -160,12 +164,11 @@ class Gene extends React.Component {
   renderStickyRefineSection = () => {
     if (!this.showingArtworksSection) { return null }
     const topMargin = this.state.showingStickyHeader ? 0 : HeaderHeight
-
     return (<View style={{ backgroundColor: 'white'}}>
         <Separator style={{marginTop:topMargin}} />
-        <View style={{flexDirection: 'row', justifyContent: 'space-between', height: 32, marginTop:8, marginBottom:8, paddingLeft: this.commonPadding, paddingRight: this.commonPadding}} >
-          <SerifText>1,xxx, works</SerifText>
-          <WhiteButton text="REFINE" style={{ marginTop:8, height: 20, width: 90}} onPress={this.refineTapped}/>
+        <View style={{flexDirection: 'row', justifyContent: 'space-between', height: 26, marginTop:12, marginBottom:12, paddingLeft: this.commonPadding, paddingRight: this.commonPadding }} >
+          <SerifText style={{ fontStyle: 'italic', marginTop:4 }}>{ this.artworkQuerySummaryString() }</SerifText>
+          {/* <WhiteButton text="REFINE" style={{ height: 26, width: 80, fontSize: 12 }} onPress={this.refineTapped}/> */}
         </View>
         <Separator/>
```

Comment:
will do a polish run pretty soon on the UI, once I've got it all hooked up in Eigen - should clean this mess up ^